packages/api/src/db/memberPlan.ts

```diff
@@ -0,0 +1,77 @@
+import {RichTextNode} from '../graphql/richText'
+import {ConnectionResult, InputCursor, Limit, SortOrder} from './common'
+
+export enum PaymentPeriodicity {
+  Monthly = 1,
+  Quarterly = 3,
+  Yearly = 12
+}
+
+export interface MemberPlan {
+  readonly id: string
+  readonly createdAt: Date
+  readonly modifiedAt: Date
+  readonly label: string
+  readonly imageID?: string
+  readonly description: RichTextNode[]
+  readonly isActive: boolean
+  readonly availablePaymentPeriodicity: PaymentPeriodicity[]
+  readonly minimumDuration: number // in months
+  readonly forceAutoRenewal: boolean
+  readonly pricePerMonthMinimum: number
+  readonly pricePerMonthMaximum: number
+  //readonly availablePaymentMethods: string[]
```

Comment:
Is there a particular reason that this is uncommented?

Reply:
Yes because i haven't implemented it yet 😄